lighthouse-core/test/fraggle-rock/scenarios/stringified-replay-test-pptr.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {promises as fs} from 'fs';
+import {promisify} from 'util';
+import {execFile} from 'child_process';
+
+import {stringify} from '@puppeteer/replay';
+
+import {LH_ROOT} from '../../../../root.js';
+import LighthouseStringifyExtension from '../../../fraggle-rock/replay/stringify-extension.js';
+import {getAuditsBreakdown, createTestState} from './pptr-test-utils.js';
+import {readJson} from '../../test-utils.js';
+
+const execFileAsync = promisify(execFile);
```

Comment:
looking at usages, could just import execFileSync

Reply:
Easier to get the `stdout` and `stderr` this way.

Reply:
ah true. lgtm then

promisify is funky it how it does this special-case transformation

> If there is an `original[util.promisify.custom]` property present, `promisify`will return its value, see `Custom promisified functions`.

<img width="656" alt="image" src="https://user-images.githubusercontent.com/4071474/180100709-16da83dc-b10d-438b-a5a4-7c5cf3a49837.png">

<img width="476" alt="image" src="https://user-images.githubusercontent.com/4071474/180100787-3a12e7d7-5329-4e11-b2b9-478dd0542d79.png">